packages/back-end/src/services/experiments.ts

```diff
@@ -527,6 +543,16 @@ export function getSnapshotSettings({
         settingsForSnapshotMetrics,
         metricOverrides: experiment.metricOverrides,
         decisionFrameworkSettings: experiment.decisionFrameworkSettings,
+        phaseLookbackWindow:
+          experiment.type === "holdout" && phase.lookbackStartDate
+            ? {
+                value: differenceInHours(
+                  phase.dateStarted,
+                  phase.lookbackStartDate
+                ),
```

Comment:
Ensure difference is positive (i.e. lookback start date is in the past).  Minimum value of 1